### PR TITLE
Covid 19 vax content update

### PR DESCRIPTION
**Pull request by stephenBarrs**
## Description
updates to content per https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/17150

## Testing done
Manual review

## Screenshots
image
image
image
image

## Acceptance criteria
- [ ] All content updates are complete

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/coronavirus-vaccination/config/uiSchema.js
Comment (ncksllvn) [inline src/applications/coronavirus-vaccination/config/uiSchema.js:47]: In the `'ui:description'` fields,  a `<p>` might give us a little bit of top/bottom margin (I can't tell if it needs that or not though so your call)

**Review by ncksllvn — APPROVED** [review]
LGTM!

File: src/applications/coronavirus-vaccination/config/uiSchema.js
Comment (stephenBarrs) [reply src/applications/coronavirus-vaccination/config/uiSchema.js:47]: @DanielleThierryUSDSVA thoughts on spacing in the Note: info for SSN and DOB?

**Review by batemapf — APPROVED** [review]
Approving this for cto-health-products to unblock you when you are ready to merge, but @DanielleThierryUSDSVA has the final approval.

**Comment by stephenBarrs** [comment]
related slack thread: https://dsva.slack.com/archives/C01FNSTQD7H/p1607612091413000

**Review by DanielleThierryUSDSVA — COMMENTED** [review]
Images looked good, and commits look good to me. Just the spacing question - see Martha's comments in slack. Thanks!

**Pull request merged**
